working/2364 - primary constructors/feature-specification.md

```diff
@@ -202,21 +202,29 @@ enum E {
 class const Point(int x, int y);
 
 enum E(String s) { one('a'), two('b') }
+```
```

Comment:
In line 173 above, it says
> The modifier `final` on a parameter in a primary constructor has no meaning
> for the parameter itself, because there is no scope where the parameter can
> be accessed. 

That is no longer true if we allow asserts (or field initializers or super-constructor invocations).

Reply:
Right, adjusted the description.